package.json

```diff
@@ -21,6 +21,7 @@
   },
   "dependencies": {
     "@hapi/joi": "^17.1.1",
+    "@types/uuid": "^8.0.0",
```

Comment:
types should always be a devDependency

Reply:
Damn, `npm install @types/uuid` automatically put it in there and didn't realise